src/Pages/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -662,16 +662,16 @@ const AddContent = () => {
                     </FormGroup>
                     <FormGroup fieldId='enable_module_hotfixes'>
                       <Switch
-                        label='Modularity filtering disabled'
-                        labelOff='Modularity filtering enabled'
-                        ouiaId={`module_hotfixes_switch_${moduleHotfixesEnabled ? 'on' : 'off'}`}
+                        label='Modularity filtering enabled'
+                        labelOff='Modularity filtering disabled'
+                        ouiaId={`module_hotfixes_switch_${modularityFilteringEnabled ? 'on' : 'off'}`}
                         aria-label='enable_module_hotfixes'
                         hasCheckIcon
                         id={'module-hotfixes-switch' + index}
                         name='module-hotfixes-switch'
-                        isChecked={moduleHotfixesEnabled}
+                        isChecked={modularityFilteringEnabled}
                         onChange={() => {
-                          updateVariable(index, { moduleHotfixesEnabled: !moduleHotfixesEnabled });
+                          updateVariable(index, { modularityFilteringEnabled: !modularityFilteringEnabled });
                         }}
                       />
                       <Tooltip content='Optional: Selecting this will set the module_hotfixes flag on the clients, allowing the repository to not be filtered by modularity'>
```

Comment:
Hi @xbhouse and @jlsherrill . I suggest the following:
```suggestion
                      <Tooltip content='When enabled, the modularity filtering prevents package updates on this repository'>
```

Reply:
But its only filtering updates for the packages contained in the module. So, your suggestion I think should be:

'When enabled, modularity filtering prevents updates for packages contained within a module in this repository'

No need article because we mention "modularity filtering" as concept here, not a specific instance of it.

Reply:
changed tooltip text to this suggestion: 'When enabled, modularity filtering prevents updates for packages contained within a module in this repository'